README.md

```diff
@@ -2,105 +2,99 @@
-live Prometheus charts for each HTTP/RPC handler, database method, or other
-piece of application logic.
+[Learn more about Autometrics here](https://autometrics.dev/).
 
-Autometrics for TypeScript provides a wrapper function and a decorator that can
-create Prometheus or OpenTelemetry metrics for your functions and class methods
-throughout your code base, as well as a language service plugin that will write
-corresponding Prometheus queries for you.
-
-See [Why Autometrics?](https://github.com/autometrics-dev#why-autometrics) for
-more details on the ideas behind autometrics
-
-![AutometricsTS demo](./assets/autometrics-ts-demo.gif)
 
 ## Features
 
-- ✨ `autometrics` wrapper instruments any function or class method to track its
+- ✨ `autometrics()` wrapper / `@Autometrics()` instruments any function or
+class method to track its
   most useful metrics
-- 🌳 Works in NodeJS and browser environments (Deno and serverless support coming soon)
+- 🌳 Works in NodeJS, experimental support for browser and Deno environments
 - 💡 Writes Prometheus queries so you can understand the data generated without
   knowing PromQL
 - 🔗 Injects links to live Prometheus charts directly into each function's doc
+- 🔍 Helps you to [identify commits](https://docs.autometrics.dev/typescript/adding-version-information) that introduced errors or increased latency
+- 🚨 Allows you to [define alerts](https://docs.autometrics.dev/typescript/adding-alerts-and-slos) using SLO best practices directly in your source code
   comments
-- [📊 Grafana dashboards](#dashboards) work out of the box to visualize the performance of instrumented functions & SLOs
-- 🚨 Generates Prometheus alerting rules using SLO best practices
-  from simple annotations in your code
+- 📊 [Grafana
+dashboards](https://github.com/autometrics-dev/autometrics-shared#dashboards)
+work out of the box and visualize the performance of instrumented functions &
+SLOs
 - ⚡ Minimal runtime overhead
 
-## How it works
+## Example 
 
-The Autometrics library:
+```typescript
+import { autometrics } from "autometrics";
 
-- Automatically instruments any wrapped function with OpenTelemetry metrics
-- Uses a Prometheus Exporter to write metrics to a `/metrics` endpoint (by
-  default on port `:9464`) or pushes them to a specified gateway (if used in
-  browser)
-- Uses a TypeScript plugin / VSCode extension to automatically add useful Prometheus queries in the doc comments for instrumented functions
+const createUserWithMetrics = autometrics(async function createUser(payload: User) {
+  // ...
+});
+
+createUserWithMetrics();
+```
+
+![AutometricsTS demo](./assets/autometrics-ts-demo.gif)
 
 ## Quickstart
 
-```shell
+1. Install the library:
+
+```bash
 npm install --save autometrics
+# or
+yarn add --save autometrics
+# or
+pnpm add --save autometrics
 ```
 
-Use the library in your code:
+2. Instrument your code using the `autometrics` wrapper or `Autometrics`
```

Comment:
It might be worth showing a tiny example of using the decorator on a minimal class